core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -203,6 +203,10 @@ package object config {
   private[spark] val HISTORY_UI_MAX_APPS =
     ConfigBuilder("spark.history.ui.maxApplications").intConf.createWithDefault(Integer.MAX_VALUE)
 
+  private[spark] val UI_SHOW_CONSOLE_PROGRESS = ConfigBuilder("spark.ui.showConsoleProgress")
```

Comment:
While updating the PR, I noticed that `internal/config/package.scala` looks more natural for this.

Like [spark.ui.retainedTasks](https://github.com/apache/spark/pull/19061/files#diff-6bdad48cfc34314e89599655442ff210R198), I added here.

Reply:
Please add document to this config using `.doc(...)`

Reply:
Sure!